spec/tasks/delete_items_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+require 'rake'
+describe "delete_items rake task " do 
+  before :all do
+    Rake.application.rake_require "tasks/sevendays"
```

Comment:
Consider renaming the task file from `sevendays.rake` to something that isn't coupled to the number of days, e.g. `todos.rake`. If the number of days changes to make todos live for 10 days say, then the `sevendays` filename will add extra work and/or be misleading.